app/templates/custom-elements/shutdown-wait.html

```diff
@@ -0,0 +1,75 @@
+<template id="shutdown-wait-template">
+  <style>
+    @import "css/spinner.css";
+
+    .overlay {
+      display: none;
+      text-align: center;
+      -webkit-touch-callout: none;
+      -webkit-user-select: none;
+      -khtml-user-select: none;
+      -moz-user-select: none;
+      -ms-user-select: none;
+      user-select: none;
+    }
+
+    :host([show="true"]) .overlay {
+      display: block;
+      position: fixed;
+      width: 100%;
+      height: 100%;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      background-color: rgba(0, 0, 0, 0.8);
+      z-index: 2;
+    }
+
+    #shutdown-wait-panel > div {
+      background-color: rgb(252, 236, 223);
+      border: 1px solid rgb(139, 97, 62);
+      max-width: 800px;
+      margin: 100px auto 0rem auto;
+      padding: 2rem;
+    }
+  </style>
+  <div id="shutdown-wait-panel" class="overlay">
+    <div>
+      <h3 id="shutdown-wait-message"></h3>
+      <div class="spinner"></div>
+    </div>
+  </div>
+</template>
+
+<script>
+  (function () {
+    const doc = (document._currentScript || document.currentScript)
+      .ownerDocument;
+    const template = doc.querySelector("#shutdown-wait-template");
+
+    customElements.define(
+      "shutdown-wait",
+      class extends HTMLElement {
+        constructor() {
+          super();
+        }
+        connectedCallback() {
+          this.attachShadow({ mode: "open" });
+          this.shadowRoot.appendChild(template.content.cloneNode(true));
+        }
+        get show() {
+          return this.getAttribute("show") === "true";
+        }
+
+        set show(newValue) {
+          this.setAttribute("show", newValue);
+        }
+
+        static get observedAttributes() {
+          return ["show"];
+        }
```

Comment:
We actually don't need to observe this attribute. I included this by mistake in `<shutdown-dialog>`, so this made me realize I need to cut it:

https://github.com/mtlynch/tinypilot/pull/212